src/frontend/shortlist/src/components/layout/profile/UserPreference.vue

```diff
@@ -0,0 +1,96 @@
+<script>
+import PreferenceDemo from "../../preferences/PreferenceDemo.vue";
+import axios from "axios";
+
+export default {
+  components: {
+    PreferenceDemo,
+  },
+  data() {
+    return {
+      prefs: {},
+      prefupdateflag: false,
+    };
+  },
+  methods: {
+    updateflag() {
+      this.prefupdateflag = !this.prefupdateflag;
+    },
+  },
+  mounted() {
+    axios
```

Comment:
emit up to view then post to keep current file fomat

Reply:
Nice note for flow understanding to communicate with API.